dotCMS/src/main/java/com/dotcms/experiments/business/web/ExperimentWebAPIImpl.java

```diff
@@ -28,28 +31,47 @@
 import javax.servlet.http.HttpServletResponse;
 import net.bytebuddy.utility.RandomString;
 
+import org.jetbrains.annotations.NotNull;
+
+
 /**
  * Default implementation of {@link ExperimentWebAPI}
  */
 public class ExperimentWebAPIImpl implements ExperimentWebAPI {
 
     @Override
-    public List<SelectedExperiment> isUserIncluded(final HttpServletRequest request, final HttpServletResponse response)
+    public SelectedExperiments isUserIncluded(final HttpServletRequest request,
+            final HttpServletResponse response, final List<String> idsToExclude)
             throws DotDataException, DotSecurityException {
+
         final List<Experiment> experimentRunning = APILocator.getExperimentsAPI()
                 .getRunningExperiment();
```

Comment:
this should be named "getRunningExperiments" in plural. Also the variable "experimentRunning" should be "experimentsRunning" or "runningExperiments"

Reply:
done https://github.com/dotCMS/core/pull/23410/commits/5c71b55498c12d6f61a334898b5aaec4f968d9f2

done https://github.com/dotCMS/core/pull/23410/commits/1ae4364cc2de81cefd382676d1e076f53ea02a6b#diff-df4c3ee11b473e0f4e91d5be7a6107bf6cdb4ab40bf0661a182dd298e00e5602L47